examples/example_unrolling_service/example_without_bazel.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""This script demonstrates how the Python example service without needing
+to use the bazel build system.
+
+Prerequisite:
+    # In the repo's INSTALL.md, follow the 'Building from source using CMake' instructions with `-DCOMPILER_GYM_BUILD_EXAMPLES=ON` added to the `cmake` command
+    # Then copy the `loop_unroller` binary
+    $ cd <path to source directory>/examples
+    $ cp <path to build directory>/examples/example_unrolling_service/loop_unroller/loop_unroller ./example_unrolling_service/loop_unroller/
```

Comment:
@ChrisCummins here as a hack I am copying the executable from the build directory to next to the Python file in the source directory. 
What should be the proper way of doing this? Creating a `py_binary` for the Python script in CMake and adding the executable as a dependency? I would prefer a way that would treat the script as a standalone.

Maybe add the path to the executable to the `PATH` environment variable?

Reply:
If you make a `py_binary` it will create a symlink in the build directory under the same subdirectory as is the source file relative to the top source directory.